frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -51,16 +56,22 @@ export const TicketDetailsCard = () => {
 	const defaultView = ticket?.properties?.[AdditionalProperties.DEFAULT_VIEW];
 	const currentIndex = filteredTickets.findIndex((tckt) => tckt._id === ticket._id);
 	const initialIndex = useRef(currentIndex);
+	const disableCycleButtons = currentIndex > -1 ? filteredTickets.length < 2 : filteredTickets.length < 1;
+
+	const getUpdatedIndex = (delta: number) => {
+		let index = currentIndex === -1 ? initialIndex.current : currentIndex;
+		if (currentIndex === -1 && delta === IndexChange.UP) index -= 1;
+		return (index + delta) % filteredTickets.length;
+	};
 
-	const changeTicketIndex = (delta: number) => {
-		const index = currentIndex === -1 ? initialIndex.current : currentIndex;
-		const updatedId = filteredTickets.slice((index + delta) % filteredTickets.length)[0]._id;
+	const changeTicketIndex = (delta: IndexChange) => {
+		const updatedId = filteredTickets.slice(getUpdatedIndex(delta))[0]._id;
```

Comment:
`getUpdatedIndex(delta)` returns the index of the ticket, so I was wondering why not just writing `filteredTickets[getUpdatedIndex(delta)]` instead of calling `slice` and get the first element.

Then I found out that, when you go to the last ticket from the first one, `getUpdatedIndex` returns `-1`. (Supposing this is why you call `slice`) I think that, if you want to keep `getUpdatedIndex` as a separate method, the latter should never return `-1` as that's not a real index. Otherwise, if you merge the 2 methods, there is an array method called `at` which is similar to the `[<index>]` operator, but accepts negative numbers. Negative numbers act the same way as python, moving backwards e.g. `[0,1,2,3,4].at(-1) => 4 `